hibernate-core/src/main/java/org/hibernate/resource/transaction/backend/jdbc/internal/JdbcResourceLocalTransactionCoordinatorImpl.java

```diff
@@ -84,8 +84,7 @@ public void explicitJoin() {
 
 	@Override
 	public boolean isJoined() {
-		return physicalTransactionDelegate != null && physicalTransactionDelegate.getStatus() == TransactionStatus.ACTIVE;
-
+		return physicalTransactionDelegate != null && getTransactionDriverControl().isActive( true );
```

Comment:
More I just wanted to make sure it was consistent.  Really all JPA says about a transaction being _active_ is "whether a resource transaction is in progress".  So it comes down to an interpretation of _in progress_.  Transactions marked for rollback-only are definitely a gray area in these discussions.  

And again my question is really more about `EM#isJoinedToTransaction` and what makes sense.  Yes it has to comply with the spec, but here all the spec says is that an EM is _joined to a transaction_ unless "the entity manager is not joined to the current transaction or if no transaction is active".  

That has 2 parts.  First, is there an "active transaction"?  That is clearly interpretive in the case of rollback-only.  The second part is about the entity manager being joined to the transaction.  The EM was joined prior to rollback-only being set, so that really boils down to whether marking a transaction as rollback-only "unjoins" the EM from the transaction.  No where does the JPA spec hint at that, that I have seen.

In my mind, `EM#isJoinedToTransaction` ought to return true after calling `#setRollbackOnly`.  Which means that `EM#getTransaction()#isActive` would need to return true.  Hibernate users can still directly check the `org.hibernate.resource.transaction.spi.TransactionStatus`